pkg/registry/styles/projects.less

```diff
@@ -19,6 +63,15 @@ table.projects-listing {
         cursor: pointer;
     }
 
+    tbody.inner-project-listing-add {
+        border-bottom: 0px;
+    }
+
+    tr.inner-project-listing {
+        display:block;
+        display: table-row;
```

Comment:
There are two conflicting display attributes. Could you what this does and why it's necessary?